### ISSUE TITLE
Trim whitespace in browser name?

### ISSUE THREAD

**Issue by dherges** [karma-runner/karma#2875]
### Expected behaviour

Should whitespace in a browser name be trimmed?

The stupid developers like me tend to do typos a lot. Recently, I put a whitespace in the browser name which went unnoticed for a few hours until the error was resolved. Of course, I was googling all sorts of unrelated error messages along the way.

### Actual behaviour

With `karma.conf.js` and `karma-chrome-launcher`:
```js
  browsers: [ 'ChromeHeadless '];
```

Then obviously it fails with:

```
Cannot load browser "ChromeNoSandbox ": it is not registered! Perhaps you are missing some plugin?
```

### Environment Details

- Karma version (output of `karma --version`): `Karma version: 1.7.1`
- Relevant part of your `karma.config.js` file: see above

### Steps to reproduce the behaviour

See above

**Comment by johnjbarton** [comment]
Seems reasonable, send a pull request.

**Comment by EzraBrooks** [comment]
Fixed by #2876.
